Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/administration-new/contests/AdministrationContests.tsx

```diff
@@ -166,6 +166,7 @@ const AdministrationContestsPage = () => {
                   ? Number(contestId)
                   : null}
               isEditMode={isEditMode}
+              skipGettingContest={isEditMode}
```

Comment:
If it's edit mode we skip. Before the logic was `skip: !isEditMode`

Reply:
!isEditMode is correct . Fixed